### ISSUE TITLE
Unable to construct class containing private (not marked with @jsonize) types.

### ISSUE THREAD

**Issue by SerialVelocity** [rcorre/jsonizer#17]
Hey,

I'm unable to construct this class:

``` d
class A {
  mixin JsonizeMe;

  private int a;

  @jsonize
  public this(int a) {
    this.a = a;
  }
}
```

**Comment by SerialVelocity** [comment]
It complains about const, it complains about it not being public and it complains about something else I think

**Comment by rcorre** [comment]
Huh. This one should definitely work. I'll take a look.

**Comment by rcorre** [comment]
Should be resolved by same fix as #20.
